docs/pages/router/advance/router-settings.mdx

```diff
@@ -1,21 +1,19 @@
 ---
 title: Router Settings
-description: Learn about upcoming and experimental features for Expo Router.
+description: Learn how to configure layouts with static properties in Expo Router.
 hideTOC: true
 sidebar_title: Settings
 ---
 
 import { APIBox } from '~/components/plugins/APIBox';
 import { FileTree } from '~/ui/components/FileTree';
 
-> **warning** This guide refers to upcoming Expo Router features which are experimental, not covered by semantic versioning and you may need to be using the latest version on `main`.
+> **Warning** `unstable_settings` currently do not work with [async routes](/router/reference/async-routes/) (development-only). This is why the feature is designated _unstable_.
```

Comment:
```suggestion
> **warning** **Warning:** `unstable_settings` currently do not work with [async routes](/router/reference/async-routes/) (development-only). This is why the feature is designated _unstable_.
```

The first `**warning**` is to indicate that this callout is of warning type. To say the word warning, we'll have to explicitly mention it. (I'm not sure if that's the case here or not).